src/main/java/com/myhome/controllers/dto/PaymentDto.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 Prathab Murugan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.myhome.controllers.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class PaymentDto {
+  private String paymentId;
+  private float charge;
```

Comment:
Money should be used with `BigDecimal`
https://stackoverflow.com/a/8148773/5785871